Classes/Helper/Helper.php

```diff
@@ -46,6 +46,8 @@ class Helper implements SingletonInterface
      *
      * @param int $uid the uid value of a tt_content record
      *
+     * @todo: move this in a repository
```

Comment:
Only usage (two times) is within the ShortcutPreviewRender. 

Could move towards that class directly. Otherwise, this helper is literally a repository aka stateless service already - so disagreeing with that comment in that form.

Todo on the class level to think about moving/renaming the whole class (albeit beeing posible as I would not consider that as public api even not marked internal).

Reply:
My purpose was to having database related methods bundled in one class, no matter about the naming, but `Repository` as name would certainly be understood easily.

Reply:
comments have been removed in 188625fc